src/oic/oic/provider.py

```diff
@@ -1000,7 +1000,7 @@ def _collect_user_info(self, session, userinfo_claims=None):
         except UnicodeEncodeError:
             try:
                 logger.debug("user_info_response: {}".format(info.encode("utf-8")))
-            except Exception:
+            except Exception:  # nosec
                 pass
```

Comment:
This is a very weird construction... I do believe that we can get rid of that completely if we do `logger.debug("...", info)` as I believe the info should contain unicode already?

Reply:
It should. And if it doesn't it is a bug in the "userinfo" method passed into the Provider